conf/messages.en

```diff
@@ -10,25 +10,22 @@ pageTitle.personal.accountDetails = Personal Account Details
 pageTitle.business.accountDetails = Business Account Details
 
 pageTitle.accountExistsIndeterminate = Check the account details
-content.accountExistsIndeterminate = We have not been able to check the account details $bankName$. Make sure the details you entered are correct.
+warning.accountExistsIndeterminate =
+content.accountExistsIndeterminate = We have not been able to check the account details with <bank name>. Make sure the details you entered are correct.
```

Comment:
Should that be $bankName$ instead of <bank name>

Reply:
It's just my placeholder, as I'd do to indicate dynamic content. Please feel free to change the way we indicate this.